applications/nrf_desktop/configuration/nrf52840dk_nrf52840/prj_mcuboot_smp.conf

```diff
@@ -126,8 +119,4 @@ CONFIG_DESKTOP_LOG=y
 # Bootloader Configuration
 
 CONFIG_BOOTLOADER_MCUBOOT=y
-
-CONFIG_STREAM_FLASH=y
-CONFIG_IMG_MANAGER=y
-CONFIG_MCUBOOT_IMG_MANAGER=y
```

Comment:
question from other pr which is a bit related to this: https://github.com/nrfconnect/sdk-nrf/pull/10999#discussion_r1189586768

Reply:
Answered under the linked comment